### PR TITLE
Update examples using new framework

### PR DESCRIPTION
**Pull request by edgarriba**
List of examples: https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples

- [x] camera_calibration: https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples/camera_calibration
- [x] camera_settings: show how to get and set the camera settings.
- [x] camera_client: https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples/camera_client
- [x] camera_pipeline -> remove
- [x] file_converter: https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples/file_converter
    - deprecate the snapshot
- [x] file_reader: https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples/file_reader
- [x] file_reader_can: https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples/file_reader_can
  - update with the new protos `AmigaTpdo1` , `AmigaPdo2`
  - https://github.com/farm-ng/farm-ng-amiga/blob/amiga-v6-protos/protos/farm_ng/canbus/amiga_v6.proto#L29-L47
- [x] motor_states_stream: https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples/motor_states_stream
- [x] vehicle_twist: https://github.com/farm-ng/farm-ng-amiga/tree/main/py/examples/vehicle_twist
  - update how to use the canbus request_reply_handle to send signals to the event service

### New ones
- [x] `camera_pointcloud`: show how to compute a pointcloud using the camera calibration and the disparity
- [x] `service_client`: hello world for creating a dummy service / client with EventServiceGrpc
- [x] `service_counter`: how to create a service with extra concurrent tasks
- [x] `service_propagation`: more complex example simulating a distributed multi-agent system interaction with a database via intermediate supervisor nodes.
- [ ] recorder_service

-------------
- [x] Needs to be merged: https://github.com/farm-ng/farm-ng-core/pull/161
- [ ] Need to update online tutorials: https://github.com/farm-ng/amiga-dev-kit/pull/190

### CODE REVIEW

File: py/examples/file_reader/utils.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        events_index (list[EventLogPosition]): [description]
+
+    Returns:
+        dict[str, list[EventLogPosition]]: [description]
+    """
+    events_dict: dict[str, list[EventLogPosition]] = defaultdict(list)
+    for event_index in events_index:
+        # NOTE: this is how we get the service name from the event uri
+        # TODO: this should be done by the event service
+        service_name = event_index.event.uri.query.split("&")[-1].split("=")[-1]
```
Comment (edgarriba) [inline py/examples/file_reader/utils.py:35]: @ethanrublee i thought we fixed that from the event service side?

File: py/examples/people_detection/README.md [hunk 2]
```diff
@@ -1,57 +0,0 @@
-# farm-ng-amiga people detection example
```
Comment (Hackerman342) [inline py/examples/people_detection/README.md:1]: Why deleting this example @edgarriba ?

File: py/examples/service_client/client.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
+# Copyright (c) farm-ng, inc.
```
[the remark this answers is not in the document]
Comment (Hackerman342) [reply py/examples/service_client/client.py:1]: Maybe we can follow up with documentation. It's better to get this merged sooner for the update of the existing examples

File: py/examples/people_detection/README.md (hunk 2 above)
Comment (edgarriba) [reply py/examples/people_detection/README.md:1]: this service doesn't exist anymore

Comment (Hackerman342) [reply py/examples/people_detection/README.md:1]: :facepalm: ofc

**Pull request merged**